src/awkward/_v2/contents/listarray.py

```diff
@@ -1506,3 +1506,9 @@ def _layout_equal(self, other, index_dtype=True, numpyarray=True):
             and self.stops.layout_equal(other.stops, index_dtype, numpyarray)
             and self.content.layout_equal(other.content, index_dtype, numpyarray)
         )
+
+    def toListOffsetArray64(self):
+        return self.to_list_offset_array()
+
+    def toRegularArray(self):
+        return self.to_regular_array()
```

Comment:
Why is this here? Is this left over from trying to apply this change in stages?

Reply:
I've left this here because I don't know what kind of deprecation period we want for internal APIs. If you're happy to make the switch then these can all go.